_core/smart/clarityRef.md

```diff
@@ -18,7 +18,7 @@ The `get-block-info` function fetches property details for a block at a specifie
 (get-block-info time 10) ;; Returns 1557860301
 ```
 
-Because the Clarity language is in pre-release, the block properties that are fetched are simulated properties from a SQLite database. The available property names are:
+Because the Clarity language is in early release, the block properties that are fetched are simulated properties from a SQLite database. The available property names are:
```

Comment:
I think we should just remove this whole "Block properties" section. It's not really helpful to a developer, and it's definitely not relevant to the language reference of Clarity.

Reply:
done!